integrations/Axeptio/browser.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+import { NAME } from "./constants";
+import logger from "../../utils/logUtil";
```

Comment:
We should update the new `logger` here. You can take reference from amplitude.

Reply:
Updated